analyzers/tests/SonarAnalyzer.UnitTest/Rules/TypeMemberVisibilityTest.cs

```diff
@@ -27,22 +27,27 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class TypeMemberVisibilityTest
     {
+        private static readonly VerifierBuilder Builder = new VerifierBuilder<TypeMemberVisibility>();
```

Comment:
Should not be static
```suggestion
        private readonly VerifierBuilder builder = new VerifierBuilder<TypeMemberVisibility>();
```